fs/dafsclient/rmtclient.cpp

```diff
@@ -391,7 +391,8 @@ void flushDaFsSocket(ISocket *socket)
         char buf[1024];
         for (;;)
         {
-            Sleep(1000);    // breathe
+            if (!totread)
+                Sleep(1000);    // breathe
```

Comment:
this means it sleeps for 1 sec. every call to flushDaFsSocket, because sleep unconditionally called the 1st time,
Shouldn't it only sleep if nothing read the 2nd time onwards?

Reply:
ok, sure.  I thought it was in the flushDaFsSocket loop sleeping each iteration whilst reading in a large buffer.
This would prevent that, sleeping only once.

Reply:
I'll move the sleep out of the loop - perhaps even eliminate it

Reply:
yeah, I don't know why the sleep's there all. We should try to avoid default delays like this if at all possible.
If it could ever go into a tight loop, it makes some sense, but readtms with its 1 minute timeout should mean it doesn't go into a tight loop.